content/docs/2.14/operate/prometheus.md

```diff
@@ -31,6 +30,18 @@ The KEDA Operator exposes Prometheus metrics which can be scraped on port `8080`
 - `keda_internal_metricsservice_grpc_server_handling_seconds` - Histogram of response latency (seconds) of gRPC that had been application-level handled by the server.
 - Metrics exposed by the `Operator SDK` framework as explained [here](https://sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/#metrics).
 
+#### Deprecated metrics
+
+The following metrics are exposed as well, but are deprecated and will be removed in a future release.
+
+- `keda_scaler_metrics_latency` - Replaced by `keda_scaler_metrics_latency_seconds`. The latency of retrieving current metric from each scaler.
+- `keda_scaler_errors` - Replaced by `keda_scaler_detail_errors_total`. The number of errors that have occurred for each scaler.
+- `keda_scaler_errors_total` - Replaced by `keda_scaler_detail_errors_total`. The total number of errors encountered for all scalers.
+- `keda_scaled_object_errors` - Replaced by `keda_scaled_object_errors_total`. The number of errors that have occurred for each ScaledObject.
+- `keda_resource_totals` - Replaced by `keda_resource_handled_total`. Total number of KEDA custom resources per namespace for each custom resource type (CRD).
+- `keda_trigger_totals` - Replaced by `keda_trigger_handled_total`. Total number of triggers per trigger type.
+- `keda_internal_scale_loop_latency` - Replaced by `keda_internal_scale_loop_latency_seconds`. Total deviation (in milliseconds) between the expected execution time and the actual execution time for the scaling loop. This latency could be produced due to accumulated scalers latencies or high load. This is an internal metric.
```

Comment:
General suggestion - Can we put the description first and then use ` (Replaced by <metric-name>)` format instead please?

This is easier to ready to know what it tells, before what it reports